examples/kafka_message_consumer_service/kafka_message_consumer_service.description

```diff
@@ -0,0 +1,2 @@
+// Ballerina natively supports Kafka. Here is an example in which we create a
```

Comment:
Same comment as above regarding the use of the phrase "natively supports"

Reply:
Fix in all the places

Reply:
Makes sense. Changed the comment.